java/arcs/android/storage/service/IMuxedStorageService.aidl

```diff
@@ -27,9 +28,11 @@ interface IMuxedStorageService {
      *
      * @param encodedStoreOptions a byte array encoding of a {@link
      *     arcs.android.storage.StoreOptionsProto}
-     * @param callback invoked whenever the storage channel responds with a
-     *     message
+     * @param callback invoked when storage channel has been created
+     * @param callback invoked whenever the storage channel responds with a message
      */
     oneway void openMuxedStorageChannel(
-        in byte[] encodedStoreOptions, IStorageChannelCallback callback);
+        in byte[] encodedStoreOptions,
+        IStorageChannelCallback callback,
```

Comment:
Might be nice to call this `channelCallback` or something, just to further disambiguate it from `messageCallback`